### DISCUSSION TITLE
Symbols panel shows enum members in wrong way

### DISCUSSION THREAD

**Discussion by fatih-erikli** [community/community#56526] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have such a typescript enum definition:

```
enum CubeFaceName {
  Front,
  Right,
  Left,
  Bottom,
  Back,
  Top,
}
```
Github shows it in a way that each item is the child of previous enum member. Probably a parsing mistake related with typescript files.

Screenshot

<img width="342" alt="Screen Shot 2023-05-27 at 04 10 32" src="https://github.com/community/community/assets/92447202/d26566f4-d569-4eeb-b544-4d3d65696577">

**Comment by Jake7208** [comment]
I don't believe they're nested based on the code you sent is it just the graphical or interface?
if not I might just be confused (could be wrong take info with a grain of salt )

**Reply by fatih-erikli** [reply]
Try it. It's reproducible in any ts file.

<img width="994" alt="Screen Shot 2023-05-29 at 05 05 45" src="https://github.com/community/community/assets/92447202/4f03bc38-bb94-43b0-85c7-d26cf4479e26">
